ministryofjustice/laa-apply-for-criminal-legal-aid
# Basic index page

## Description of change
Adds the index page "Crime applications" so the user can view applications.

Note this is a "simple" table at the moment as it is following govuk rather than MoJ styles for now.

Also adds a `govuk_formatted_date` helper method and a `full_name` method to person (this one could be moved to a presenter if its cleaner.

## Link to relevant ticket
[CRIMAP-80](https://dsdmoj.atlassian.net/browse/CRIMAP-90)
## Notes for reviewer
I wanted to keep this small so all it does is render things at the moment, further PRs will add more functionality:

- "Make new applications" will actually start an application
- At some point delete will work
- Statuses will be reflect an application's status (currently hard coded)
- Sorting of page
- Pagination (later)

There are also patterns that could be introduced that aren't included in this PR, for instance:
- Using presenters for the progress tag (this already exists but will need a little refactoring to work in this context rather than the Task list)


## Screenshots of changes (if applicable)

### After changes:
<img width="987" alt="Screenshot 2022-08-17 at 11 02 29" src="https://user-images.githubusercontent.com/13377553/185092358-348bfe4c-7f37-4725-86d2-b1b649860983.png">


## How to manually test the feature
- Create an application
- got to `/crime_applications`
- You should see the table with the list of cases
- if you click a case it will got to the its task list.
